kitchen-azurerm.gemspec

```diff
@@ -2,7 +2,7 @@ lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 Gem::Specification.new do |spec|
   spec.name          = "kitchen-azurerm"
-  spec.version       = "0.14.8"
+  spec.version       = "0.14.9"
```

Comment:
Please could you revert this as we already have a 0.14.9 - the version number will get added at release time